modules/logging-release-notes-5-9-10.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging_release_notes/logging-5-9-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-9-10_{context}"]
+= Logging 5.9.10
+
+This release includes link:https://access.redhat.com/errata/RHSA-2024:10990[{logging-uc} {for} Bug Fix Release 5.9.10].
+
+[id="logging-release-notes-5-9-10-bug-fixes_{context}"]
+== Bug Fixes
+
+* Before this update, Loki attempted to detect the level of log messages, which caused confusion when the collector also detected log levels and produced different results. With this update, automatic log level detection in Loki is disabled. (link:https://issues.redhat.com/browse/LOG-6321[LOG-6321])
+
+* Before this update, when the `ClusterLogForwarder` custom resource defined `tls.insecureSkipVerify: true` in combination with `type: http` and an HTTP URL, the certificate validation was not skipped. This misconfiguration caused the collector to fail because it attempted to validate certificates despite the setting. With this update, when `tls.insecureSkipVerify: true` is set, the URL is checked for the HTTPS. An HTTP URL will cause a misconfiguration error. (link:https://issues.redhat.com/browse/LOG-6376[LOG-6376])
+
+* Before this update, when any infrastructure namespaces were specified in the application inputs in the `ClusterLogForwarder` custom resource, logs were generated with the incorrect `log_type: application` tags. With this update, when any infrastructure namespaces are specified in the application inputs, logs are generated with the correct `log_type: infrastructure` tags. (link:https://issues.redhat.com/browse/LOG-6377[LOG-6377])
++
+[IMPORTANT]
+====
+When upgrading to {logging-uc} {for} 5.9.10, if you previously added any infrastructure namespaces in the application inputs in the `ClusterLogForwarder` custom resource, you must add the permissions for collecting logs from infrastructure namespaces. For more details, see "Setting up log collection".
```

Comment:
Update is the preferred term per the [term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update):
```suggestion
When updating to {logging-uc} {for} 5.9.10, if you previously added any infrastructure namespaces in the application inputs in the `ClusterLogForwarder` custom resource, you must add the permissions for collecting logs from infrastructure namespaces. For more details, see "Setting up log collection".
```